platform/default/src/mbgl/gfx/headless_backend.cpp

```diff
@@ -0,0 +1,16 @@
+#include <mbgl/gfx/headless_backend.hpp>
+
+namespace mbgl {
+namespace gfx {
+
+HeadlessBackend::HeadlessBackend(Size size_)
```

Comment:
do we need some asserts on the given `size_`?

Reply:
No need, lower layers of API allow 0.